Header header
nav_msgs/Odometry robot_odom
nav_msgs/Odometry goal_odom
mav_msgs/RateThrust action
bool use_vehicle_frame
